understanding/21/pointer-cancellation.html

```diff
@@ -9,28 +9,43 @@
-<p>There is a distinction between when someone touches a screen and when they remove their finger. Similarly, in mouse interaction, there is a difference between pressing and releasing the mouse button. When activation occurs only when the pointer is released, users have the opportunity to prevent (cancel) the activation. They can move their finger or other pointer (e.g., the mouse cursor) away from the target when they realise that they touched that target inadvertently or erroneously.</p>     
-        <p>Up-event activation refers to the activation of a target when the pointer is released. In a touchscreen interaction, when the finger touches a target, the up-event activation ony occurs when the finger is lifted while still being within the target boundary. Similarly in mouse interaction, the up-event occurs when the mouse button is released while the cursor is still within the boundary of the initial target when the mouse button was pressed.</p>
-      <p>Authors can reduce the problem of users inadvertently triggering an action by using generic platform activation/click events that activate functionality on the up-event. For example, the "click" event in JavaScript triggers on release of the primary mouse button, and is an example of an implicit up-event. Despite its name, the onclick event handler works both for mouse interaction as well as for touch interaction.</p>
-      
-      <p>If activation on the down-event is necessary, there are several options:</p>
+			<p>The intent of this success criterion is to make it easier for users to prevent accidental or erroneous pointer input. People with various disabilities can inadvertently initiate touch or mouse events with unwanted results. Each of the following subsections roughly aligns with the bullets of this Success Criterion, and outlines a means of allowing users to cancel pointer operations.</p>
+			<h3>Up-Event activation or completion</h3>
+			<p>The most accessible way to incorporate pointer cancellation is to make activation occur on the up-event.</p>
+     
+        <p>Up-event activation refers to the activation of a target when the pointer is released. In a touchscreen interaction, when the finger touches a target, the up-event activation only occurs when the finger is lifted while still being within the target boundary. Similarly in mouse interaction, the up-event occurs when the mouse button is released while the cursor is still within the boundary of the initial target set when the mouse button was pressed.</p>
+      <p>Authors can reduce the problem of users inadvertently triggering an action by using generic platform activation/click events that activate functionality on the up-event. For example, the "click" event in JavaScript triggers on release of the primary mouse button, and is an example of an implicit up-event. Despite its name, the onclick event handler is device-independent and also works for touch and keyboard interaction.</p>
```

Comment:
tiny nitpick: might be better to talk about the "`click` event is device independent", rather than just the "onclick event handler" (as click is of course also device independent when attached via `addEventListener("click", ...)`

Reply:
@patrickhlauke, I am not enough of a developer to grasp the nuances, but it sounds like maybe we can use both?  
Does this make sense?
> Despite the names, both the <code>click</code> event and the <code>onclick</code> event handler are device-independent and also work for touch...

Reply:
i don't really think you need to mention `onclick` per se at all? as it's simply the handler if you were doing old-school "event handlers as attributes in HTML" (a very discouraged practice) or doing something like `foo.onclick = ....` in JavaScript which is also quite oldschool compared to using `addEventListener`, since `onclick` would only allow a single function to be tied?

Reply:
So just:
> Despite the name, the <code>click</code> event is device-independent and also works for touch...

Reply:
yup :)